Weapon swap bug fix

ringo23333 opened the pull request:
Changed a few words in ui update script
don't know why theres 2 commits, other is probably some settings thing

ringo23333 committed:
(IN UI UPDATE) changed weapon ui switch

changed weapon ui script from getkey to getkeydown

ringo23333 committed:
Merge branch 'main' into ringo2

Reallife101 reviewed (changes requested):
When I switch weapons with mouse wheel, the crosshair doesn't update.

ringo23333 committed:
Merge branch 'main' into ringo2

ringo23333 committed:
Weapon ui fix

Got rid of updateUI script and moved it to WeaponSwap to make it work with the scroll wheel too
